docs/modules/general/pages/implementation-details/mfe-content-loading/webcomponents.adoc

```diff
@@ -0,0 +1,195 @@
+
+[IMPORTANT]
+====
+Using **@onecx/angular-webcomponents** library according to the guide is the recommended way of exposing Angular content. The set of functionalities provided via the library is taking care of all requirements related to the webcomponents adaptation. Apart from automatic registration to Custom Elements Registry, they ensure that the following issues are covered:
+
+* Every module and Remote Component is going to have its own router. Routers (Shell's router, displayed module's router and router of each displayed Remote Component) need to be synchronized so each can react on the router events.
+* Single Angular version can only create one platform to operate on, which means that for every content loaded within a page the same platform has to be used for a specific version.
+* ngZone of Shell UI has to be shared with all modules and Remote Components to allow Web Components to function correctly.
+
+In depth explanations for those issues can be found in the https://www.angulararchitects.io/en/blog/multi-framework-and-version-micro-frontends-with-module-federation-the-good-the-bad-the-ugly/[module-federation multi-framework and version micro frontends document].
+
+====
+
+==  OneCX adaptation to webcomponents method
+As stated in xref:implementation-details/mfe-content-loading/index.adoc[Microfrontend content loading documentation], every method of Microfrontend's content exposal has an impact on the OneCX platform behavior for both module and Remote Component loading. The adaptations made by OneCX to work with Web Components were briefly mentioned in the Web Components preview, however depending on the content type the behavior varies.
+
+// TODO: Add link to example of migrating angular modules to integrate with OneCX
+=== Module loading behavior
+Shell UI App is responsible for displaying Microfrontend's module when router navigation is matched for a desired Microfrontend. With webcomponents method Shell UI will adapt the loadChildren method to:
+
+. load desired remote module via module-federation (no change)
+. use **WebComponentLoaderModule** as the module to be lazy-loaded (instead of the Microfrontend's module)
+
+The https://angular.dev/guide/ngmodules/lazy-loading[loadChildren method bounded to a Route] is an asynchronous method that in the end should return a module to be displayed when activating a certain Route. With webcomponents method, the desired Microfrontend's module is still being loaded using module-federation. During that loading, it should register a component in Custom Elements Registry as mentioned in previous section. For OneCX platform it means that it no longer should load the Microfrontend's module directly, but ensure that the html code has correct element in order to display the module.
```

Comment:
... but ensure that the html code has correct **Custom Element(?)** in order to display the module.